lib/onceover/testconfig.rb

```diff
@@ -236,6 +237,19 @@ def write_spec_helper(location, repo)
       )
     end
 
+    def copy_spec_files(repo)
+      source_files = @included_specs.map { |glob| Dir.glob "#{repo.spec_dir}/#{glob}" }.flatten.uniq
+      source_files.each do |source_file|
+        target_file = File.join(repo.tempdir, 'spec', source_file.delete_prefix(repo.spec_dir))
```

Comment:
`String#delete_prefix` was [added in Ruby 2.5](https://github.com/ruby/ruby/commit/10082360b9124c3eaabfccf4fe10a3640c40a05e) but the test suite is run against Ruby 2.4 and 2.5.

Ruby 2.4 [has reached EOL](https://en.wikipedia.org/wiki/Ruby_(programming_language)#Table_of_versions) a few months ago, maybe the CI build matrix can be updated to test against supported Ruby versions?

Reply:
Hum, Ruby 2.4 is shipped with Puppet Agent 5…  Maybe it's better to keep support for Ruby 2.4 and use good old `String#sub`:

```suggestion
        target_file = File.join(repo.tempdir, 'spec', source_file.sub(/^#{repo.spec_dir}/, ''))
```